helios-client/src/main/java/com/spotify/helios/common/descriptors/DeploymentGroup.java

```diff
@@ -199,6 +222,9 @@ public Builder setName(final String name) {
       return this;
     }
 
+    // TODO(negz): Annotate this as @JsonProperty("job")?
+    // AFAICT we're currently serialising this with the key "jobId", despite annotating it as
+    // @JsonProperty("job") in the constructor and javadoc.
```

Comment:
also I think this comment is at the wrong place in the file, not in the middle of the builder class?